homeassistant/components/generic/camera.py

```diff
@@ -63,6 +64,7 @@
         ),
         vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
         vol.Optional(CONF_RTSP_TRANSPORT): vol.In(RTSP_TRANSPORTS.keys()),
+        vol.Optional(CONF_USE_WALLCLOCK_AS_TIMESTAMPS, default=False): cv.boolean,
```

Comment:
I don't think we need to add this option to the YAML config during the deprecation period.

Reply:
Sure, I'll take it out then. BTW I am not a fan of removing yaml, but I am in the minority 👎